imap_processing/utils.py

```diff
@@ -101,17 +101,22 @@ def convert_raw_to_eu(dataset: xr.Dataset, conversion_table_path, packet_name):
     return dataset
 
 
-def create_dataset(packets: list[Packet], met_name="shcoarse") -> xr.Dataset:
+def create_dataset(
+    packets: list[Packet], met_name="shcoarse", include_header=True, skip_keys=None
```

Comment:
I wonder if something like `spacecraft_time_key` would be better / more explicit than `met_name`. When I look at `met_name` it is hard to tell what it is used for.

Reply:
yes. I appreciate better name suggestions because I was not happy with what I had.